harness.sh

```diff
@@ -238,7 +245,7 @@ y
 ${MASTER_WALLET_SEED}
 EOF" C-m
 sleep 1
-tmux send-keys "dcrwallet -C mwallet.conf " C-m
+tmux send-keys "dcrwallet -C mwallet.conf --debuglevel=info" C-m
```

Comment:
`info` is already the default log level, we dont need to set the flag here.

Reply:
explicitly stating that there like the rest of the components, makes it easy to switch debug levels as well by editing that.